Decred Wallet/view_controller/ReceiveViewController.swift

```diff
@@ -92,6 +109,32 @@ class ReceiveViewController: UIViewController,UIDocumentInteractionControllerDel
         self.present(alertController, animated: true, completion: nil)
     }
     
+    // MARK: - Private instance methods
+
+    private func toggleView() {
+        let syncsCount: Int? = Settings.readOptionalValue(for: Settings.Keys.SyncsCount)
+        if isNewWalletSetup && !isSynced {
```

Comment:
This can simplify to 
```
if isSynced || isNewWalletSetup {
    ... generate address etc
} else {
    ... show sync in progress label
}
```

Also, `isSynced` and `isNewWalletSetup` can be declared as local variables in this function, rather than as global variables. Not sure they're needed elsewhere.